test/lib/agent_helper.js

```diff
@@ -632,17 +622,6 @@ helper.isSupportedVersion = function isSupportedVersion(version) {
   return semver.gt(process.version, version)
 }
 
-/**
- * The https-proxy-server we support finally supports keep alive
- * See: https://github.com/TooTallNate/proxy-agents/pull/147
- * In order for tap to shutdown we must destroy the https agent.
- * This assumes the agent already exists as a singleton so we can destroy
- * the active http agent
- */
-helper.destroyProxyAgent = function destroyProxyAgent() {
-  require('../../lib/collector/http-agents').proxyAgent().destroy()
-}
```

Comment:
With the changes in this PR, there are no longer any references to this method. So, goodbye.